python/lsst/pipe/base/cmdLineTask.py

```diff
@@ -573,8 +573,9 @@ def parseAndRun(cls, args=None, config=None, log=None, doReturnResults=False):
               `lsst.pipe.base.ArgumentParser.parse_args` method.
             - ``taskRunner``: the task runner used to run the task (an instance of `Task.RunnerClass`).
             - ``resultList``: results returned by the task runner's ``run`` method, one entry per invocation.
-                This will typically be a list of `None` unless ``doReturnResults`` is `True`;
-                see `Task.RunnerClass` (`TaskRunner` by default) for more information.
+                This will typically be a list of `Struct` containing at least an
+                ``exitStatus`` member; see `Task.RunnerClass` (`TaskRunner` by
```

Comment:
While I agree this more accurately describes what is in the parseAndRun return struct, it would be better still if this explicitly said exitStatus is either 0 or 1. Suggestion: "This will typically be a list of `Struct`, each containing at least one ``exitStatus`` int (0 or 1 for success or failure); see `Task.RunnerClass` (`TaskRunner` by default) for more details."